blockwork/containers/container.py

```diff
@@ -453,6 +459,13 @@ def launch(
                 target = tmpdir / "tmp" / file.name
                 logging.debug(f"Copying {file} to {target}")
                 shutil.copy(file, target)
+            # Create a temporary script to execute
+            # NOTE: Arguments containing spaces will be wrapped by double quotes
+            run_script = tmpdir / "tmp" / "run.sh"
+            run_script.write_text(
+                "#!/bin/bash\n" + " ".join((f'"{x}"' if " " in x else x) for x in command) + "\n"
+            )
```

Comment:
Can you use https://docs.python.org/3/library/shlex.html#shlex.join here? This join is pretty error prone